src/cli/constant.js

```diff
@@ -443,10 +443,19 @@ const detailedOptionMap = detailedOptions.reduce(
   {}
 );
 
+const apiDetailedOptionMap = detailedOptions.reduce(
+  (current, option) =>
+    option.forwardToApi && option.forwardToApi !== option.name
```

Comment:
Isn't forwardToApi `true` in a lot of places?

Reply:
At this point they're already normalized to the camelcased strings per https://github.com/taion/prettier/blob/70313e67950edb34a8cab0911c5b3ab1a90b7db6/src/cli/constant.js#L425